frontend/src/app/(navfooter)/preset/page.tsx

```diff
@@ -0,0 +1,51 @@
+"use client"
```

Comment:
Fully client rendered pages should be avoided :-1:

Reply:
Removed